standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/HMSHandler.java

```diff
@@ -75,22 +78,7 @@
 import java.lang.reflect.UndeclaredThrowableException;
 import java.nio.ByteBuffer;
 import java.security.PrivilegedExceptionAction;
-import java.util.AbstractMap;
```

Comment:
nit: can you restore the imports?

Reply:
If the import is not used, it is better to remove it (Sonar/CheckStyle do too).